app/src/main/res/layout/session_layout_land_large.xml

```diff
@@ -24,20 +24,18 @@
         <ImageView
             android:id="@+id/session_no_video_view"
             android:layout_width="wrap_content"
-            android:layout_height="16sp"
-            android:layout_marginLeft="5dp"
+            android:layout_height="27.5sp"
             android:adjustViewBounds="true"
-            android:baseline="15sp"
+            android:scaleType="fitStart"
             android:src="@drawable/ic_novideo" />
 
         <ImageView
             android:id="@+id/session_bell_view"
             android:layout_width="wrap_content"
-            android:layout_height="16sp"
-            android:layout_marginLeft="5dp"
+            android:layout_height="27.5sp"
```

Comment:
The bell icon adjusts to the height (without the red bar) of the camera icon next to it by adding `android:padding="4dp"` here. I think this looks a bit better.

:heavy_check_mark: Tested on:

- Pixel 6, Android 13, portrait and landscape modes
- 10" tablet, Android 13 in landscape and landscape split mode.